proto/mixer.proto

```diff
@@ -214,6 +214,12 @@ message GetStatsRequest {
   // The dcid of the StatisticalVariable.
   string stats_var = 2;
 
+  // The measurement method of the observation.
```

Comment:
please specify that these are optional here.

Reply:
Done